### ISSUE TITLE
legalinsurrection.com

### ISSUE THREAD

**Issue by terrorist96** [uBlockOrigin/uAssets#835]
### URL(s) where the issue occurs

`https://legalinsurrection.com/2010/12/the-license-plate-would-have-been-enough/`

### Describe the issue

Image blocked

### Notes

This filter is causing it: `|http://$third-party,image,domain=100percentfedup.com|allthingsvegas.comgopocalypse.org|beforeitsnews.com|legalinsurrection.com|newser.com|quirlycues.com|thegatewaypundit.com|viralnova.com|wakingtimes.com|wnd.com|xtribune.com`

**Comment by okiehsch** [comment]
@gorhill this issue is caused by EasyList fixing "Yavli" sites, uBO-filters already fixes that issue for all sites included in the above mentioned EasyList filter, so in my opinion, the best fix for this issue would be to disable the EasyList filter.
````
|http://$third-party,image,badfilter,domain=100percentfedup.com|allthingsvegas.comgopocalypse.org|beforeitsnews.com|legalinsurrection.com|newser.com|quirlycues.com|thegatewaypundit.com|viralnova.com|wakingtimes.com|wnd.com|xtribune.com
````
The alternative would be to use an exception filter for the bocked image request.
`@@||blogspot.com^$image,domain=legalinsurrection.com`

**Comment by gorhill** [comment]
Is this sort of false positive common with that filter?

If this is one isolated case, I say we leave it alone, not worth to start creating an exception filter if it's an isolated case. For now I rather not badfilter that filter, we just can't verify that no case of unblocked content slipping through is going to occur as a result of disabling that filter, it affects too many pages on too many sites.

For such probably rare non-crippling false positives, I rather let EasyList maintainers deal with the outcome of their own filter as they see fit.

**Comment by terrorist96** [comment]
I'm not sure how common it is. I only noticed this one instance. I'll ping @ryanbr if he wants to fix it in Easylist (though me pinging him has rarely produced a result in the past).

**Comment by gorhill** [comment]
Closing as wontfix for now. Can be re-opened if ever this sort of issue become common.
